README.md

```diff
@@ -97,14 +97,14 @@ The following environment variables are supported:
-| __PUPPET_STORECONFIGS__                    | Sets `storeconfigs` in puppet.conf<br><br>`true`                                                                                                              |
-| __PUPPET_STORECONFIGS_BACKEND__            | Sets `storeconfigs_backend` in puppet.conf<br><br>`puppetdb`                                                                                                  |
+| __OPENVOX_REPORTS__                         | Sets `reports` in puppet.conf<br><br>`puppetdb`                                                                                                               |
+| __OPENVOX_STORECONFIGS__                    | Sets `storeconfigs` in puppet.conf<br><br>`true`                                                                                                              |
+| __OPENVOX_STORECONFIGS_BACKEND__            | Sets `storeconfigs_backend` in puppet.conf<br><br>`puppetdb`                                                                                                  |
 | __OPENVOXSERVER_MAX_ACTIVE_INSTANCES__      | The maximum number of JRuby instances allowed<br><br>`1`                                                                                                      |
 | __OPENVOXSERVER_MAX_REQUESTS_PER_INSTANCE__ | The maximum HTTP requests a JRuby instance will handle in its lifetime (disable instance flushing)<br><br>`0`                                                 |
 | __OPENVOXSERVER_JAVA_ARGS__                 | Arguments passed directly to the JVM when starting the service<br><br>`-Xms1024m -Xmx1024m`                                                                   |
-| __USE_PUPPETDB__                           | Whether to connect to puppetdb<br>Sets `PUPPET_REPORTS` to `log` and `PUPPET_STORECONFIGS` to `false` if those unset<br><br>`true`                            |
-| __PUPPETDB_SERVER_URLS__                   | The `server_urls` to set in `/etc/puppetlabs/puppet/puppetdb.conf`<br><br>`https://puppetdb:8081`                                                             |
+| __USE_OPENVOXDB__                           | Whether to connect to puppetdb<br>Sets `OPENVOX_REPORTS` to `log` and `OPENVOX_STORECONFIGS` to `false` if those unset<br><br>`true`                            |
+| __OPENVOXDB_SERVER_URLS__                   | The `server_urls` to set in `/etc/puppetlabs/puppet/puppetdb.conf`<br><br>`https://puppetdb:8081`                                                             |
 | __PUPPETDB_HOSTNAME__                      | The DNS name of the puppetdb <br><br> Defaults to `puppetdb`                                                                                                  |
 | __PUPPETDB_SSL_PORT__                      | The TLS port of the puppetdb <br><br> Defaults to `8081`                                                                                                      |
```

Comment:
Are those two still correct?

Reply:
Actually, it looks like neither of those is in use anywhere, so they probably can be dropped.
```
$ git grep -nE '(OPENVOX|PUPPET)DB_(HOSTNAME|SSL_PORT)'
README.md:108:| __PUPPETDB_HOSTNAME__                      | The DNS name of the puppetdb <br><br> Defaults to `puppetdb`                                                                                                  |
README.md:109:| __PUPPETDB_SSL_PORT__                      | The TLS port of the puppetdb <br><br> Defaults to `8081`                                                                                                      |
```

Reply:
yeah, i have an issue for checking all the variables.
https://github.com/voxpupuli/crafty/issues/83

Reply:
@jaevans can you drop these two?